### PR TITLE
fix targetedAction error for fireball/issues/4129

### PR DESCRIPTION
**Pull request by knoxHuang**
Re: cocos-creator/fireball#4129

Changes proposed in this pull request:
- 

> Makes sure these boxes are checked before submitting your PR - thank you!
> - [ ] If your pull request has gone "stale", you should **rebase** your work on top of the latest version of the upstream branch.
> - [ ] If your commit history is full of small, unimportant commits (such as "fix pep8" or "update tests"), **squash** your commits down to a few, or one, discreet changesets before submitting a pull request.
> - To official teams:
>   - [ ] Check that your javascript is following our [style guide](https://github.com/cocos-creator/fireball/blob/dev/.github/CONTRIBUTING.md) and end files with a newline
>   - [ ] Document new code with comments in source code based on [API Docs](https://github.com/cocos-creator/fireball#api-docs)

@cocos-creator/engine-admins

### CODE REVIEW

File: jsb/jsb-action.js [hunk 1]
```diff
@@ -112,6 +112,11 @@ for (var i = 0; i < actionArr.length; ++i) {
     }
 }
 
+cc.targetedAction = function (target, action) {
+    target._sgNode._owner = target;
+    return new cc.TargetedAction(target._sgNode, action);
```
Comment (pandamicro) [inline jsb/jsb-action.js]: 用户如果用 new cc.TargetedAction 还是会有问题，需要重写 cc.TargetedAction.prototype._ctor

在 jsb_create_api.js 中有原始实现

File: jsb/jsb-action.js [hunk 2]
```diff
@@ -123,31 +128,25 @@ cc.Follow.prototype.update = function(dt) {
     }
 };
 
-cc.Show.prototype.update = function (dt) {
-    var target = this.getTarget();
+function changeRenderEnabled (toggle, target, state) {
```
Comment (jareguo) [inline jsb/jsb-action.js]: 建议定义成 function setRendererVisibility (sgNode, toggleVisible, visible)

File: jsb/jsb-action.js [hunk 3]
```diff
@@ -112,6 +112,11 @@ for (var i = 0; i < actionArr.length; ++i) {
     }
 }
 
+cc.targetedAction = function (target, action) {
+    target._sgNode._owner = target;
```
Comment (jareguo) [inline jsb/jsb-action.js]: runAction 时，不是本来就会设置 _owner 吗？

Comment (knoxHuang) [reply jsb/jsb-action.js]: 但是这里不走 runAction

Comment (pandamicro) [reply jsb/jsb-action.js]: 这一行可以删除了，在 _ctor 里面已经做了

**Comment by jareguo** [comment]
LGTM

File: jsb/jsb-action.js [hunk 4]
```diff
@@ -112,6 +112,16 @@ for (var i = 0; i < actionArr.length; ++i) {
     }
 }
 
+cc.targetedAction = function (target, action) {
+    return new cc.TargetedAction(target, action);
+};
```
Comment (knoxHuang) [inline jsb/jsb-action.js]: 好像这里应该就不需要在写一次了

**Comment by pandamicro** [comment]
LGTM

**Pull request merged**